src/Spice86.Core/Emulator/CPU/CfgCpu/Parser/InstructionParser.cs

```diff
@@ -0,0 +1,309 @@
+namespace Spice86.Core.Emulator.CPU.CfgCpu.Parser;
+
+using Spice86.Core.Emulator.CPU.CfgCpu.ParsedInstruction;
+using Spice86.Core.Emulator.CPU.CfgCpu.ParsedInstruction.Instructions;
+using Spice86.Core.Emulator.CPU.CfgCpu.ParsedInstruction.Instructions.JmpNearImm;
+using Spice86.Core.Emulator.CPU.CfgCpu.ParsedInstruction.Instructions.MovMoffsAcc;
+using Spice86.Core.Emulator.CPU.CfgCpu.ParsedInstruction.Instructions.MovRmImm;
+using Spice86.Core.Emulator.CPU.CfgCpu.ParsedInstruction.Instructions.MovRmReg;
+using Spice86.Core.Emulator.CPU.CfgCpu.ParsedInstruction.Instructions.PushPopF;
+using Spice86.Core.Emulator.CPU.CfgCpu.ParsedInstruction.Prefix;
+using Spice86.Core.Emulator.CPU.CfgCpu.Parser.SpecificParsers;
+using Spice86.Core.Emulator.CPU.Registers;
+using Spice86.Core.Emulator.Memory.Indexable;
+using Spice86.Shared.Emulator.Memory;
+
+using System.Linq;
+
+public class InstructionParser  : BaseInstructionParser {
+    private readonly Grp1Parser _grp1Parser;
+    private readonly MovRegImmParser _movRegImmParser;
+    private readonly OperationOnRegIndexParser _incRegParser;
+    private readonly OperationOnRegIndexParser _decRegParser;
+    private readonly OperationOnRegIndexParser _pushRegParser;
+    private readonly OperationOnRegIndexParser _popRegParser;
+    private readonly Grp45Parser _grp45Parser;
+    private readonly AluOperationParser _addAluOperationParser;
+    private readonly AluOperationParser _orAluOperationParser;
+    private readonly AluOperationParser _adcAluOperationParser;
+    private readonly AluOperationParser _sbbAluOperationParser;
+    private readonly AluOperationParser _andAluOperationParser;
+    private readonly AluOperationParser _subAluOperationParser;
+    private readonly AluOperationParser _xorAluOperationParser;
+    private readonly AluOperationParser _cmpAluOperationParser;
+
+    
+    public InstructionParser(IIndexable memory, State state) : base(new(memory), state) {
+        _grp1Parser = new(this);
+        _movRegImmParser = new(this);
+        _incRegParser = new(this, "IncReg");
```

Comment:
those strings could be public or private consts string at the class level.

Reply:
Replaced by moxy